packages/braid-design-system/src/lib/components/Button/Button.docs.tsx

```diff
@@ -242,6 +349,7 @@ const docs: ComponentDocs = {
           </Stack>,
         ),
     },
+    ...choosingRightButtonDoc,
```

Comment:
Any particular reason not to just inline these docs?

Reply:
Yep. Given they are multiple sections that fall under the same heading, wanted to ensure no one ever accidentally adds or re-orders the the docs page and interleaves sections.